pages/index.js

```diff
@@ -141,11 +167,16 @@ const Home = ({ latestStories }) => {
 
   const [caseData, setCaseData] = useState({})
   const [cumulativeCase, setCumulativeCase] = useState(null)
+  const [cumulativeTest, setCumulativeTest] = useState(null)
 
   const [cumulativeData, setCumulativeData] = useState({})
   const [totalCases, setTotalCases] = useState(null)
   const [totalCasesDate, setTotalCasesDate] = useState(null)
   
+  const commaNumber = (number) => {
```

Comment:
would prefer renaming the function to be more grammatically correct + descriptive, it's fine if the name is longer especially since it's only being used in one place